hack/go-lint.sh

```diff
@@ -6,4 +6,5 @@ podman run --rm \
     --volume "${PWD}:/go/src/github.com/openshift/installer:z" \
     --workdir /go/src/github.com/openshift/installer \
     docker.io/golangci/golangci-lint:v1.48.0 \
-    golangci-lint run "${@}"
+    golangci-lint run "${@}" --new-from-rev=75173a17cf
+    #golangci-lint run "${@}" -c .golangci.yaml
```

Comment:
We don't want to add this here I think. You can run `./hack/go-lint.sh --new-from-rev=<commit>`. 
Happy to discuss in a new PR if you want. The idea was that devs might want to slowly work on the existing linting issues but for CI there was no way to solve all of them at once to make the test pass.

Reply:
🤦 I did not mean to commit this. Good catch @r4f4...removing..